.github/workflows/test-and-build.yml

```
@@ -58,8 +58,8 @@ jobs:
 
       - name: Fetch GAMMA
         run: |
-          aws s3 cp s3://hyp3-software/GAMMA_SOFTWARE-20230712_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz .
-          tar -zxvf GAMMA_SOFTWARE-20230712_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz
+          aws s3 cp s3://hyp3-software/GAMMA_SOFTWARE-20231208_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz .
+          tar -zxvf GAMMA_SOFTWARE-20231208.linux64_ubuntu2204.tar.gz
```

```suggestion
          tar -zxvf GAMMA_SOFTWARE-20231208_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz
```
Bah.

> download: s3://hyp3-software/GAMMA_SOFTWARE-20231208_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz to ./GAMMA_SOFTWARE-20231208_ISP_DIFF_LAT.linux64_ubuntu2204.tar.gz
tar (child): GAMMA_SOFTWARE-20231208.linux64_ubuntu2204.tar.gz: Cannot open: No such file or directory